package.json

```diff
@@ -50,7 +50,6 @@
     "ember-load-initializers": "1.0.0",
     "ember-resolver": "4.5.0",
     "ember-scroll-to": "0.6.4",
-    "ember-source": "~2.15.0",
```

Comment:
Why ember was removed?

Reply:
Wow, I don't think that's me and did not see it. Sorry.